resourcematch-app/src/components/TextInputForm.tsx

```diff
@@ -20,6 +20,9 @@ const Button = styled.button`
   margin: 10px;
   min-width: 95px;
   height: 35px;
+  color: black;
+  background-color: #87CEEB
+  ;
 `
 
 const InputLabel = styled.label`
```

Comment:
Putting this here as a general note: I noticed a couple of things on the form submission:

1) The form field itself doesn't get disabled (this seems to be an existing bug)
2) The button does not look disabled to me even though it is :man_shrugging: (this seems to be related to the styling update above)